CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.9)
+project(csv C)
+
+enable_testing()
+#set(CMAKE_CTEST_COMMAND ctest -V)
+set(CSV_VERSION_MAJOR 3)
+set(CSV_VERSION_MINOR 3)
+set(CSV_VERSION_PATCH 0)
+set(CSV_VERSION_STRING ${CSV_VERSION_MAJOR}.${CSV_VERSION_MINOR}.${CSV_VERSION_PATCH} )
+
+add_test(check_csv check_csv)
+add_executable(check_csv EXCLUDE_FROM_ALL  test_csv.c)
+add_custom_target(check COMMAND ${CMAKE_CTEST_COMMAND})
```

Comment:
You do not need to create a `check` target manually, ctest already creates a `test` one.